### DISCUSSION TITLE
Dark mode for marketplace

### DISCUSSION THREAD

**Discussion by noahdepalma** [community/community#4710] [Accessibility] [answered]
As far as I know, the Github Marketplace has no dark mode which is really annoying

**Comment by gaknoll** [comment, marked as answer]
Hi @TacoSnack - just finding this feedback, so I apologize for not getting you a response sooner! We recognize that marketplace not yet supporting GitHub themes is frustrating and can often lead to jarring flashes between light and dark themes. Marketplace uses a mix of design systems that prevent us from enabling GitHub themes support without a redesign. It is on our backlog and we look forward to getting this fixed.
